models/articles.models.js

```diff
@@ -28,10 +32,19 @@ exports.findArticles = (queryObj) => {
     }
   }
 
-  queryString += ` GROUP BY (article_id)
-  ORDER BY created_at DESC;`;
+  queryString += ` GROUP BY (article_id)`;
 
-  return db.query(queryString, queryValues).then(({ rows }) => {
+  if (sort_by && order) {
+    queryString += ` ORDER BY ${sort_by} ${order}`;
+  } else if (sort_by) {
+    queryString += ` ORDER BY ${sort_by} DESC`;
+  } else if (order) {
+    queryString += ` ORDER BY created_at ${order}`;
+  } else {
+    queryString += ` ORDER BY created_at DESC`;
+  }
```

Comment:
We want to make sure we are sanitising these variables before adding them directly to our sql query string. Without this we are opening ourselves up to sql injections. Any idea how we can sanitise these requests?